src/test/java/com/fasterxml/jackson/dataformat/xml/lists/ListSerializationTest.java

```diff
@@ -110,7 +110,7 @@ public void testStringList() throws IOException
     // [dataformat-xml#148]
     public void testIteratorSerialization() throws Exception
     {
-        assertEquals("<Bean148><item>2</item><item>1</item><item>0</item></Bean148>",
```

Comment:
How did this happen? Isn't this backwards-incompatible change in serialization?

Reply:
Ok, one actual request: It'd be good to have separate test class for Iterators or "Iterable types".
It'd be quite small class but I think would be cleaner.